internal/transport/http/handler.go

```diff
@@ -0,0 +1,36 @@
+package http
+
+import (
+	"github.com/gorilla/mux"
+	wallet_service "github.com/o-mercan/Wallet-Service-Api/internal/wallet"
+	log "github.com/sirupsen/logrus"
+)
+
+type Handler struct {
+	Router  *mux.Router
+	Service *wallet_service.Service
+}
+
+func NewHandler(service *wallet_service.Service) *Handler {
+	return &Handler{
+		Service: service,
+	}
+}
+
+func (h *Handler) SetUpRoutes() {
+	log.Info("Routes are setting")
+	h.Router = mux.NewRouter()
+
+	h.Router.HandleFunc("/api/v1/wallets/add", h.AddNewWallet).Methods("POST")
+	h.Router.HandleFunc("/api/v1/wallets/deposit/{id}", h.DepositWallet).Methods("POST")
```

Comment:
Eğer deposite yapmak istediğin wallet id'sini url'de göndermek istiyorsan şu şekilde kullanmak daha güzel olur.
/api/v1/wallets/{id}/deposit 

{id} parametresini wallets dan sonra eklenmeli.

Reply:
Düzelttim